src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableKeyStatTile.tsx

```diff
@@ -149,16 +149,28 @@ const EditableKeyStatTile = ({
                       />
                     </div>
                   </div>
-                  <Details summary="Guidance text" open>
+                  <Details
+                    className="govuk-!-margin-bottom-3"
+                    summary="Guidance text"
+                    open
+                  >
                     <FormFieldTextInput<KeyStatsFormValues>
                       id={`key-stat-dataDefinitionTitle-${id}`}
                       name="dataDefinitionTitle"
                       label="Guidance title"
                     />
-                    <FormFieldTextArea<KeyStatsFormValues>
-                      id={`key-stat-dataDefinition-${id}`}
+                    <FormFieldWysiwygArea
```

Comment:
Can we allow the previously mentioned toolbar prop here to also be defined on the FormFieldWysiwygArea component and be passed on to the wysiwyg